recipes/libressl/2.9.2/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibreSSLConan(ConanFile):
+    name = "libressl"
+    version = "2.9.2"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.libressl.org/"
+    license = "OpenSSL - OpenBSD"
```

Comment:
```suggestion
    license = ("OpenSSL", "BSD", "ISC")
```

```
  LibReSSL files are retained under the copyright of the authors. New
  additions are ISC licensed as per OpenBSD's normal licensing policy,
  or are placed in the public domain. 

  The OpenSSL code is distributed under the terms of the original OpenSSL
  licenses which follow:

  LICENSE ISSUES
  ==============

  The OpenSSL toolkit stays under a dual license, i.e. both the conditions of
  the OpenSSL License and the original SSLeay license apply to the toolkit.
  See below for the actual license texts.  In case of any license issues
  related to OpenSSL please contact openssl-core@openssl.org.
```